source/Nevermore/QueryBuilder.cs

```diff
@@ -44,6 +44,12 @@ public IQueryBuilder<TRecord> Where(string whereClause)
             return this;
         }
 
+        public IQueryBuilder<TRecord> WhereNull(string fieldName, bool not)
```

Comment:
I would add `WhereNotNull` instead of passing a `not` flag

Reply:
Yes, that's better